ui/StatusQ/src/modelcount.cpp

```diff
@@ -0,0 +1,29 @@
+#include "StatusQ/modelcount.h"
+
+#include <QAbstractItemModel>
+
+ModelCount::ModelCount(QObject* parent) : QObject(parent)
+{
+    auto model = qobject_cast<QAbstractItemModel*>(parent);
+
+    if (model == nullptr)
```

Comment:
Could print a warning here, in case somebody tries to attach to a non-model thing

Reply:
I was considering that but finally left without warning because built-in attached types list `ListView` or `Layout` doesn't print any warnings when attached to inadequate object.

Reply:
Fair enough